### PR TITLE
[FIX] Moved merging check from purchase order level to line level.

### PR DESCRIPTION
**Pull request by n17**

### CODE REVIEW

File: procurement_mto_analytic/models/procurement.py [hunk 1]
```diff
@@ -9,18 +9,11 @@ class ProcurementRule(models.Model):
     @api.multi
     def _prepare_purchase_order_line(self, product_id, product_qty,
                                      product_uom, values, po, supplier):
-        res = super(ProcurementRule, self).\
+        res = super(ProcurementRule, self). \
```
Comment (aisopuro) [inline procurement_mto_analytic/models/procurement.py]: Unnecessary whitespace.

File: procurement_mto_analytic/models/purchase_order_line.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    def _merge_in_existing_line(self, product_id, product_qty, product_uom, location_id, name, origin, values):
+        """ This function purpose is to be override with the purpose to forbide _run_buy  method
```
Comment (aisopuro) [inline procurement_mto_analytic/models/purchase_order_line.py]: This comment could describe what this extension does.
> Do not merge into a line with a different analytic account

**Review by aisopuro — CHANGES_REQUESTED** [review]
Just a few quibbles, nothing major.

**Comment by aisopuro** [comment]
@n17 please "redirect" this PR to be merged into the main OCA repo. That way we can have it run CI checks for us as well.

**Comment by mlaitinen** [comment]
Is this really supposed to target `11.0` of our own repository?

**Comment by n17** [comment]
> Is this really supposed to target `11.0` of our own repository?

@mlaitinen I think so. Do you think it should have been pushed directly to the OCA repository?

**Comment by mlaitinen** [comment]
> > Is this really supposed to target `11.0` of our own repository?
> 
> @mlaitinen I think so. Do you think it should have been pushed directly to the OCA repository?

I mean, we try to keep the `x.0` branches clean and without any extra commits compared to `OCA/x.0`. So I think you either want to make a PR for `avoinsystems/11.0-avoinsystems` (which doesn't exist yet but can be created by forking `11.0`) or `OCA/11.0`. Right, @aisopuro ?

**Comment by aisopuro** [comment]
Agreed @mlaitinen . `avoinsystems/*.0` should be identical to `OCA/*.0`

**Comment by n17** [comment]
The OCA pull request based on this was just merged. Can this PR be closed now?

**Comment by aisopuro** [comment]
Yes.

**Pull request closed without merging**